plugin-kbindicator/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(XKB_COMMON REQUIRED xkbcommon)
 pkg_check_modules(XKB_COMMON_X11 QUIET xkbcommon-x11)
 
+include_directories(${XKB_COMMON_INCLUDEDIR})
```

Comment:
> however using target_include_directories(${PLUGIN} ${XKB_COMMON_X11_INCLUDE_DIRS}) instead doesn't.

Of course it doesn't, because the target is defined later in the `BUILD_LXQT_PLUGIN` macro... sorry I've missed that.